bliss/datasets/galsim_galaxies.py

```diff
@@ -163,24 +213,36 @@ def _draw_pareto_flux(self, n_samples=1) -> Tensor:
         uniform_samples = torch.rand(n_samples) * u_max
         return self.min_flux / (1.0 - uniform_samples) ** (1 / self.alpha)
 
-    def draw_galaxy_params(self, n_samples=1):
-        # create galaxy as mixture of Exponential + DeVacauleurs
-        if self.flux_sample == "uniform":
-            total_flux = self._uniform(self.min_flux, self.max_flux, n_samples=n_samples)
-        elif self.flux_sample == "pareto":
-            total_flux = self._draw_pareto_flux(n_samples=n_samples)
-        else:
-            raise NotImplementedError()
-        disk_frac = self._uniform(0, 1, n_samples=n_samples)
-        beta_radians = self._uniform(0, 2 * np.pi, n_samples=n_samples)
-        disk_q = self._uniform(0, 1, n_samples=n_samples)
-        disk_a = self._uniform(self.min_a_d, self.max_a_d, n_samples=n_samples)
+    @staticmethod
+    def _gamma(concentration, loc, scale, n_samples=1):
+        x = torch.distributions.Gamma(concentration, rate=1.0).sample((n_samples,))
+        return x * scale + loc
 
-        bulge_q = self._uniform(0, 1, n_samples=n_samples)
-        bulge_a = self._uniform(self.min_a_b, self.max_a_b, n_samples=n_samples)
-        return torch.stack(
-            [total_flux, disk_frac, beta_radians, disk_q, disk_a, bulge_q, bulge_a], dim=1
-        )
+
+class GalsimGalaxyDecoder:
+    def __init__(
+        self,
+        slen,
+        n_bands,
+        background,
+        pixel_scale,  # SDSS
```

Comment:
this comment can be removed (referred to default value) 

```suggestion
        pixel_scale
```